astropy/time/tests/test_basic.py

```diff
@@ -745,6 +792,15 @@ def test_decimalyear():
                            jd0 + 0.75 * d_jd])
 
 
+def test_fits_year0():
```

Comment:
Yeah, we need to make sure we cover that otherwise software archeologists might think that we had predicted the end of the world in 10000